tests/test_interface.rb

```diff
@@ -586,8 +582,7 @@ def negotiate_auto_helper(interface, default, cmd_ref)
                  "Error: #{inf_name} negotiate auto value not #{default}")
 
     pattern = cmd_ref.test_config_get_regex[default ? 1 : 0]
-    assert_interface_match_line(interface.name, pattern,
-                                "#{inf_name} no negotiate auto cfg mismatch")
+    assert_show_match(pattern: pattern)
```

Comment:
Shouldn't the command parameter be specified?

    assert_show_match(command: cmd, pattern: pattern)